src/ansible_creator/resources/common/gitignore/__meta__.yml

```diff
@@ -0,0 +1,19 @@
+new_collection:
+  additions:
+    template: False
+    value: ""
+
+ansible_project:
+  additions:
+    template: True
+    value: |
+      .logs/*
+      *.retry
+      *.vault
+      collections/*
+      !collections/ansible_collections
+      !collections/requirements.yml
+      collections/ansible_collections/*
+      !collections/ansible_collections/{{ scm_org}}
```

Comment:
The code changes look good to me. What I'm unsure about is why we need L17 and L18. If our goal is to gitignore everything except for the scaffolded collection, shouldn't L16 and L19, be enough?

Reply:
Nevermind, I understand now.